### PR TITLE
fix: don't include unnecessary files in NPM package

### PR DESCRIPTION
**Pull request by alnr**
Ref https://github.com/ory/kratos-selfservice-ui-node/issues/239

`npm pack` reports:

```
npm notice 
npm notice 📦  @ory/kratos-selfservice-ui-node@0.11.0-alpha.2
npm notice === Tarball Contents === 
npm notice 11.4kB LICENSE                     
npm notice 3.9kB  README.md                   
npm notice 11B    lib/index.d.ts              
npm notice 3.3kB  lib/index.js                
npm notice 668B   lib/pkg/index.d.ts          
npm notice 5.0kB  lib/pkg/index.js            
npm notice 136B   lib/pkg/logger.d.ts         
npm notice 727B   lib/pkg/logger.js           
npm notice 1.1kB  lib/pkg/middleware.d.ts     
npm notice 3.2kB  lib/pkg/middleware.js       
npm notice 661B   lib/pkg/route.d.ts          
npm notice 77B    lib/pkg/route.js            
npm notice 211B   lib/pkg/sdk/index.d.ts      
npm notice 1.1kB  lib/pkg/sdk/index.js        
npm notice 690B   lib/pkg/ui.d.ts             
npm notice 5.0kB  lib/pkg/ui.js               
npm notice 100B   lib/routes/404.d.ts         
npm notice 963B   lib/routes/404.js           
npm notice 100B   lib/routes/500.d.ts         
npm notice 1.3kB  lib/routes/500.js           
npm notice 169B   lib/routes/error.d.ts       
npm notice 1.8kB  lib/routes/error.js         
npm notice 103B   lib/routes/health.d.ts      
npm notice 364B   lib/routes/health.js        
npm notice 323B   lib/routes/index.d.ts       
npm notice 1.4kB  lib/routes/index.js         
npm notice 169B   lib/routes/login.d.ts       
npm notice 8.5kB  lib/routes/login.js         
npm notice 175B   lib/routes/recovery.d.ts    
npm notice 2.2kB  lib/routes/recovery.js      
npm notice 183B   lib/routes/registration.d.ts
npm notice 4.3kB  lib/routes/registration.js  
npm notice 175B   lib/routes/sessions.d.ts    
npm notice 7.9kB  lib/routes/sessions.js      
npm notice 175B   lib/routes/settings.d.ts    
npm notice 11.6kB lib/routes/settings.js      
npm notice 104B   lib/routes/static.d.ts      
npm notice 1.2kB  lib/routes/static.js        
npm notice 183B   lib/routes/verification.d.ts
npm notice 2.4kB  lib/routes/verification.js  
npm notice 173B   lib/routes/welcome.d.ts     
npm notice 6.9kB  lib/routes/welcome.js       
npm notice 1.9kB  package.json                
npm notice 1.9kB  src/index.ts                
npm notice === Tarball Details === 
npm notice name:          @ory/kratos-selfservice-ui-node                   
npm notice version:       0.11.0-alpha.2                                    
npm notice filename:      @ory/kratos-selfservice-ui-node-0.11.0-alpha.2.tgz
npm notice package size:  19.8 kB                                           
npm notice unpacked size: 94.0 kB                                           
npm notice shasum:        8f6981bdcf477bbaf2bf760742036b744c0e4c37          
npm notice integrity:     sha512-EiywRwn6l5grc[...]DSHHoxYVNn9Cg==          
npm notice total files:   44                                                
npm notice 
ory-kratos-selfservice-ui-node-0.11.0-alpha.2.tgz

```

Does that look reasonable, @Benehiko ?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
   "license": "Apache-2.0",
   "author": "Aeneas Rekkas",
   "main": "src/index.ts",
+  "files": [
+    "lib/*"
```
Comment (Benehiko) [inline package.json:17]:
```suggestion
    "lib/*",
    "views/*"
```

Comment (Benehiko) [reply package.json:17]: I believe views should also be included

**Review by Benehiko — CHANGES_REQUESTED** [review]

**Comment by alnr** [comment]
Now we get 

```
npm notice 
npm notice 📦  @ory/kratos-selfservice-ui-node@0.11.0-alpha.2
npm notice === Tarball Contents === 
npm notice 11.4kB LICENSE                                       
npm notice 3.9kB  README.md                                     
npm notice 11B    lib/index.d.ts                                
npm notice 3.3kB  lib/index.js                                  
npm notice 668B   lib/pkg/index.d.ts                            
npm notice 5.0kB  lib/pkg/index.js                              
npm notice 136B   lib/pkg/logger.d.ts                           
npm notice 727B   lib/pkg/logger.js                             
npm notice 1.1kB  lib/pkg/middleware.d.ts                       
npm notice 3.2kB  lib/pkg/middleware.js                         
npm notice 661B   lib/pkg/route.d.ts                            
npm notice 77B    lib/pkg/route.js                              
npm notice 211B   lib/pkg/sdk/index.d.ts                        
npm notice 1.1kB  lib/pkg/sdk/index.js                          
npm notice 690B   lib/pkg/ui.d.ts                               
npm notice 5.0kB  lib/pkg/ui.js                                 
npm notice 100B   lib/routes/404.d.ts                           
npm notice 963B   lib/routes/404.js                             
npm notice 100B   lib/routes/500.d.ts                           
npm notice 1.3kB  lib/routes/500.js                             
npm notice 169B   lib/routes/error.d.ts                         
npm notice 1.8kB  lib/routes/error.js                           
npm notice 103B   lib/routes/health.d.ts                        
npm notice 364B   lib/routes/health.js                          
npm notice 323B   lib/routes/index.d.ts                         
npm notice 1.4kB  lib/routes/index.js                           
npm notice 169B   lib/routes/login.d.ts                         
npm notice 8.5kB  lib/routes/login.js                           
npm notice 175B   lib/routes/recovery.d.ts                      
npm notice 2.2kB  lib/routes/recovery.js                        
npm notice 183B   lib/routes/registration.d.ts                  
npm notice 4.3kB  lib/routes/registration.js                    
npm notice 175B   lib/routes/sessions.d.ts                      
npm notice 7.9kB  lib/routes/sessions.js                        
npm notice 175B   lib/routes/settings.d.ts                      
npm notice 11.6kB lib/routes/settings.js                        
npm notice 104B   lib/routes/static.d.ts                        
npm notice 1.2kB  lib/routes/static.js                          
npm notice 183B   lib/routes/verification.d.ts                  
npm notice 2.4kB  lib/routes/verification.js                    
npm notice 173B   lib/routes/welcome.d.ts                       
npm notice 6.9kB  lib/routes/welcome.js                         
npm notice 2.0kB  package.json                                  
npm notice 1.9kB  src/index.ts                                  
npm notice 10B    views/error.hbs                               
npm notice 319B   views/layouts/auth.hbs                        
npm notice 434B   views/layouts/settings.hbs                    
npm notice 434B   views/layouts/welcome.hbs                     
npm notice 341B   views/login.hbs                               
npm notice 280B   views/partials/fork_me.hbs                    
npm notice 153B   views/partials/messages.hbs                   
npm notice 110B   views/partials/ory_branding.hbs               
npm notice 397B   views/partials/standard_headers.hbs           
npm notice 224B   views/partials/ui_docs_button.hbs             
npm notice 272B   views/partials/ui_node_anchor.hbs             
npm notice 172B   views/partials/ui_node_image.hbs              
npm notice 378B   views/partials/ui_node_input_button.hbs       
npm notice 905B   views/partials/ui_node_input_checkbox.hbs     
npm notice 587B   views/partials/ui_node_input_default.hbs      
npm notice 86B    views/partials/ui_node_input_hidden.hbs       
npm notice 354B   views/partials/ui_node_input_password.hbs     
npm notice 511B   views/partials/ui_node_input_social_button.hbs
npm notice 291B   views/partials/ui_node_script.hbs             
npm notice 924B   views/partials/ui_node_text.hbs               
npm notice 84B    views/partials/ui_nodes.hbs                   
npm notice 451B   views/partials/ui_screen_button.hbs           
npm notice 169B   views/partials/ui.hbs                         
npm notice 39B    views/recovery.hbs                            
npm notice 354B   views/registration.hbs                        
npm notice 1.7kB  views/session.hbs                             
npm notice 763B   views/settings.hbs                            
npm notice 43B    views/verification.hbs                        
npm notice 869B   views/welcome.hbs                             
npm notice === Tarball Details === 
npm notice name:          @ory/kratos-selfservice-ui-node                   
npm notice version:       0.11.0-alpha.2                                    
npm notice filename:      @ory/kratos-selfservice-ui-node-0.11.0-alpha.2.tgz
npm notice package size:  23.6 kB                                           
npm notice unpacked size: 105.7 kB                                          
npm notice shasum:        59a7d394b7c18bf4a013cce8f49da28869001207          
npm notice integrity:     sha512-NqPRDE6hYV/7w[...]b9q82VSeEnLCA==          
npm notice total files:   73                                                
npm notice 
ory-kratos-selfservice-ui-node-0.11.0-alpha.2.tgz
```

**Review by Benehiko — APPROVED** [review]
LGTM :)

**Review by zepatrik — APPROVED** [review]

**Pull request merged**